ui/components/ReconciledObjectsTable.tsx

```diff
@@ -35,23 +35,33 @@ function ReconciledObjectsTable({
   return (
     <div className={className}>
       <DataTable
-        sortFields={["name", "type", "namespace", "status"]}
         fields={[
-          { value: "name", label: "Name" },
+          {
+            value: "name",
+            label: "Name",
+            sortType: SortType.string,
+            sortValue: ({ name }) => name,
+          },
           {
             label: "Type",
             value: (u: UnstructuredObject) => `${u.groupVersionKind.kind}`,
+            sortType: SortType.string,
+            sortValue: (u: UnstructuredObject) => `${u.groupVersionKind.kind}`,
           },
           {
             label: "Namespace",
             value: "namespace",
+            sortType: SortType.string,
+            sortValue: ({ namespace }) => namespace,
```

Comment:
No longer necessary, right?

https://github.com/weaveworks/weave-gitops/pull/1522/commits/746b1beefa0b8cf476cdad61868f09e2d0dcd5aa

Reply:
My branch was out of date :( will fix

Reply:
ALSO - noticed a problem with this - right now DataTable is using the field having a SortType as an indication of whether it's sortable or not - we need a different indication of which columns are sortable